modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/rel/set/IgniteColocatedMinus.java

```diff
@@ -43,20 +43,20 @@ public IgniteSingleMinus(
      * Constructor.
      * TODO Documentation https://issues.apache.org/jira/browse/IGNITE-15859
```

Comment:
seems all javadoc for touched classes should be rewritten )

Reply:
Done